forward_command_controller/include/forward_command_controller/forward_command_controller.hpp

```diff
@@ -72,6 +72,8 @@ class ForwardCommandController : public controller_interface::ControllerInterfac
   controller_interface::return_type update() override;
 
 protected:
+  virtual CallbackReturn read_parameters();
```

Comment:
Not related to this PR, but I just noticed: the parent class is spelled wrong. Should be `controller_interface::ControllerInterface`

Reply:
It looks like this has been fixed in the meantime.  :+1:  (Which probably broke API, but oh well)